src/Reflex/Class.hs

```diff
@@ -62,9 +62,9 @@ module Reflex.Class
   , EitherTag (..)
   , eitherToDSum
   , dsumToEither
-    -- ** Switching between 'Event's
-  , switchPromptly
-  , switchPromptOnly
+    -- ** Collapsing 'Event . Event'
+  , switchHoldNeverPromptly
+  , switchHoldNeverPromptOnly
```

Comment:
The "*Only" variant should be the only one of these that exists, or perhaps we should immediately deprecate `switchHoldNeverPromptly`.  The original implementation of `switchPromptly` was a mistake that should not be propagated further.

Reply:
Agreed to do this later separately; best not to mix up naming improvements with performance-impacting changes.